### PR TITLE
[GEOMETRY-103] - Migrate to JUnit 5

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: commons-geometry-core/src/test/java/org/apache/commons/geometry/core/GeometryTestUtils.java [hunk 1]
```diff
@@ -68,15 +68,15 @@ public static void assertThrows(final Runnable r, final Class<?> exceptionType)
     public static void assertThrows(final Runnable r, final Class<?> exceptionType, final String message) {
         try {
             r.run();
-            Assert.fail("Operation should have thrown an exception");
+            Assertions.fail("Operation should have thrown an exception");
```
Comment (darkma773r) [inline commons-geometry-core/src/test/java/org/apache/commons/geometry/core/GeometryTestUtils.java:71]: Could this entire method be replaced with the JUnit assertThrows?

Comment (arturobernalg) [reply commons-geometry-core/src/test/java/org/apache/commons/geometry/core/GeometryTestUtils.java:71]: Hi @darkma773r , 
I completely agree with you. but, what do you think to do this in a separate jira?  as a upgrade?

Comment (darkma773r) [reply commons-geometry-core/src/test/java/org/apache/commons/geometry/core/GeometryTestUtils.java:71]: A separate task sounds good.

**Review by darkma773r — APPROVED** [review]
Looks good! This is something I've been wanting to do for a while, so thank you.

**Pull request merged**